api/namex/resources/statistics/wait_time_statistics.py

```diff
@@ -0,0 +1,47 @@
+from flask import jsonify
+
+from namex.services.name_request.utils import handle_exception
+from namex.services.statistics.wait_time_statistics import WaitTimeStatsService
+from namex.services.exceptions import ApiServiceException
+from namex.utils.logging import setup_logging
+from flask_restplus import Namespace, Resource, cors, fields
+from flask_jwt_oidc import AuthError
+
+from http import HTTPStatus
+
+from namex.utils.util import cors_preflight
+
+setup_logging()  # important to do this first
+
+# Register a local namespace for the requests
+api = Namespace('waitTimeStats', description='API for Wait Time Statistics')
+
+
+@api.errorhandler(AuthError)
```

Comment:
not sure why we have this since we are not doing nay authentication. Talk to Lucas.